File: crates/deslop/tests/fixtures/fused-golden-php/shape_delta.php
<?php

class AuditDescriptor
{
    public string $ledger = 'delta-audit';
    public int $depth = 128;
    public string $archive = 'https://delta.example.com/audit';

    public function render(string $actor): string
    {
        return $this->archive . '/' . $actor . '/' . $this->ledger . '?depth=delta';
    }
}

File: crates/deslop/tests/fixtures/fused-golden-csharp/shape_delta.cs
namespace Golden.Shapes;

public sealed class AuditDescriptor
{
    public string Ledger { get; } = "delta-audit";
    public int Depth { get; } = 128;
    public string Archive { get; } = "https://delta.example.com/audit";

    public string Render(string actor)
    {
        return Archive + "/" + actor + "/" + Ledger + "?depth=" + Depth;
    }
}
